dashboard/lib/projects_list.rb

```diff
@@ -1,4 +1,15 @@
 module ProjectsList
+  # Maximum number of projects of each type that can be requested.
+  MAX_LIMIT = 100
+
+  # List of valid project types, excluding 'all' which is also valid.
+  PUBLISHED_PROJECT_TYPES = %w(
+    applab
+    gamelab
+    playlab
+    artist
+  )
```

Comment:
Currently, each "project type" here is both a "project category" that will be grouped together in the returned data, as well as the exact name of the project type stored in the database. In the future, these two concepts will diverge, when we add the "hoc" project category, which will contain the "frozen", "starwars", "gumball", "infinity", and "iceage" project types.